### PR TITLE
[Translation] Added a script to display the status of translations

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

This is the script I used to get the missing translations for each locale. I think it could come in handy for Symfony to always keep translations in sync.

### CODE REVIEW

File: src/Symfony/Component/Translation/Resources/bin/translation-status.php [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+$usageInstructions = <<<END
+
+  Usage instructions
+  ---------------------------------------------------------------------
+
+  $ cs symfony-code-root-directory/
```
Comment (pierredup) [inline src/Symfony/Component/Translation/Resources/bin/translation-status.php]: I think the `cs` here needs to be `cd`

Comment (norkunas) [inline src/Symfony/Component/Translation/Resources/bin/translation-status.php]: cd?

File: src/Symfony/Component/Translation/Resources/bin/translation-status.php [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+  # show the translation status of all locales
+  $ php translation-status.php
```
Comment (pierredup) [inline src/Symfony/Component/Translation/Resources/bin/translation-status.php]: I might be missing something here, but this file isn't in the root directory of the project, so should the full path not be added here?

**Review by nicolas-grekas — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @javiereguiluz.

**Pull request merged**